packages/wonder-blocks-dropdown/components/action-menu.test.js

```diff
@@ -55,12 +55,10 @@ describe("ActionMenu", () => {
         // Close menu with space
         opener.simulate("keydown", {keyCode: keyCodes.space});
         opener.simulate("keyup", {keyCode: keyCodes.space});
-        opener.simulate("click", {preventDefault: jest.fn()});
```

Comment:
Because these components are mounted, the `keydown`/`keyup` events trigger a click.

Reply:
I wondered about that when I first saw that.  The differences between `shallow` and `mount` a interesting.